sdk/identity/Azure.Identity/README.md

```diff
@@ -211,6 +211,7 @@ Not all credentials require this configuration. Credentials which authenticate t
 |[`ClientAssertionCredential`][ref_ClientAssertionCredential]|Authenticates a service principal using a signed client assertion. |
 |[`ClientCertificateCredential`][ref_ClientCertificateCredential]|Authenticates a service principal using a certificate. | [Service principal authentication](https://learn.microsoft.com/entra/identity-platform/app-objects-and-service-principals)
 |[`ClientSecretCredential`][ref_ClientSecretCredential]|Authenticates a service principal using a secret. | [Service principal authentication](https://learn.microsoft.com/entra/identity-platform/app-objects-and-service-principals)
+|`AzurePipelinesServiceConnectionCredential`|Supports [Microsoft Entra Workload ID](https://learn.microsoft.com/azure/devops/pipelines/release/configure-workload-identity?view=azure-devops) on Azure Pipelines.| [example](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-in-azure-pipelines-with-service-connections)
```

Comment:
This example is a JS link. Point to the sample you included in this PR.

@KarishmaGhiya Note that Chris put the credential in this "Authenticate service principals" section. Can you update the JS README accordingly? You have it in the "Authenticate Azure-hosted applications" section.

Reply:
good catch - I created a new one and forgot to update the link